http_server.py

```diff
@@ -0,0 +1,36 @@
+import socket
+
+ADDR = ("127.0.0.1", 8001)
+server = socket.socket(
+    socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_IP
+)
+
+
+def response_ok():
+    return "HTTP/1.1 200 OK <html><head><title>Success!</title></head><body><header><h1>Success!</h1></header><p>It worked</p></body></html>"
```

Comment:
I'm guessing by now you have a better idea of why this isn't right.  (same comment for `response_error` below).  If not, see me.